### ISSUE TITLE
Bug with `dropdown_input(type = "search multiple selection")`: the dropdown remains open after having no more options

### ISSUE THREAD

**Issue by daattali** [Appsilon/shiny.semantic#327]
Perhaps this is an issue with an older version of sementicUI that has since then been fixed but isn't yet in this package.

When a dropdown supports multiple selections and search, when you click with the mouse on a selection, the dropdown box remains open with nothing inside of it. Example:

```
library(shiny)

ui <- shiny.semantic::semanticPage(
  shiny.semantic::dropdown_input("test", c("a1", "a2", "b"), type = "search multiple selection")
)


server <- function(input, output, session) {}

shinyApp(ui, server)
```

Here's a GIF of what happens:

![semantic](https://user-images.githubusercontent.com/952340/102133546-41951980-3e23-11eb-8ed7-b66bb0aa0817.gif)

I tested the same type of input in the semantic UI demo page https://semantic-ui.com/modules/dropdown.html#multiple-search-selection and it doesn't have this problem.

**Comment by dokato** [comment]
Thanks @daattali for reporting these bugs, we will have a look. I actually had a first go yesterday on refactoring tabset a bit to make bring `updateTabset` functionality up. Just to let you know for future debugging, `shiny.semantic` migrated to [FomanticUI](fomantic-ui.com) at the beginning of this year as original Semantic library looks abandoned now.

**Comment by dokato** [comment]
This also should be fixed with #333.
